UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSupplies.json

```diff
@@ -0,0 +1,376 @@
+{
+  "resourceType": "Questionnaire",
+  "id": "UrologicalSupplies",
+  "name": "UrologicalSuppliesOrder",
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/UrologicalSupplies-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPatientInfo-prepopulation"
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+      "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPractitionerInfo-prepopulation"
+    }
+  ],
+  "title": "UrologicalSupplies Order Questionnaire",
+  "status": "draft",
+  "date": "2020-06-22",
+  "publisher": "Da Vinci DTR",
+  "item": [
+    {
+      "linkId": "1",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+          "valueCanonical": "questionnaire/patient-info"
+        }
+      ],
+      "type": "display"
+    },
+    {
+      "linkId": "2",
+      "type": "group",
+      "required": false,
+      "text": "Diagnosis",
+      "item": [
+        {
+          "type": "open-choice",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "drop-down",
+                    "display": "Drop down"
+                  }
+                ],
+                "text": "Drop down"
+              }
+            }
+          ],
+          "required": false,
+          "linkId": "2.1",
+          "text": "Patient Diagnosis",
+          "answerOption": [
```

Comment:
Can you prepopulate the diagnoses from CQL instead of hard-coding options here? You can take a look at how the progress note is done.